src/ECC.DanceCup.Api.Infrastructure.Storage/Migrations/00004_Add_Tournaments.cs

```diff
@@ -0,0 +1,31 @@
+using FluentMigrator;
+
+namespace ECC.DanceCup.Api.Infrastructure.Storage.Migrations;
+
+[Migration(00004, TransactionBehavior.None)]
+public class Add_Tournaments : SqlMigration
+{
+    protected override string? UpSqlCommand =>
+        """
+        create table "tournaments" (
+            "id" bigserial primary key,
+            "version" int not null,
+            "created_at" timestamp not null,
+            "changed_at" timestamp not null,
+            "user_id" bigint not null,
+            "name" text not null,
+            "description" text,
+            "date" timestamp not null,
+            "state" int not null,
```

Comment:
```suggestion
            "state" text not null,
```

Давай хранить текстом